fineract-provider/src/main/java/org/apache/fineract/infrastructure/jobs/filter/LoanCOBFilterHelper.java

```diff
@@ -43,11 +43,13 @@
 import org.apache.fineract.portfolio.loanaccount.domain.Loan;
 import org.apache.fineract.portfolio.loanaccount.domain.LoanRepository;
 import org.apache.fineract.portfolio.loanaccount.rescheduleloan.domain.LoanRescheduleRequestRepository;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.http.HttpMethod;
 import org.springframework.stereotype.Component;
 
 @RequiredArgsConstructor
 @Component
+@ConditionalOnProperty(value = "fineract.job.loan-cob-enabled", havingValue = "true")
```

Comment:
Should we have @conditional(LoanCOBEnabledCondition.class) instead? I might be wrong but Loan COB filter can be used only on manager, right?

Reply:
changed